src/Foundation/Commerce/code/Services/Catalog/ICatalogService.cs

```diff
@@ -0,0 +1,45 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Commerce.Services.Catalog
+{
+    using Models;
+
+    using Models.Entities;
+
+    /// <summary>
+    /// Performs operations with catalog
+    /// </summary>
+    public interface ICatalogService
```

Comment:
Left only this interface, let's implementation will be with "not implemented exception" for now